CMakeLists.txt

```diff
@@ -0,0 +1,423 @@
+#=============================================================================
+# Copyright 2022 NVIDIA Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#=============================================================================
+
+cmake_minimum_required(VERSION 3.20.1 FATAL_ERROR)
+
+cmake_policy(SET CMP0077 NEW)
+set(CMAKE_POLICY_DEFAULT_CMP0077 NEW)
+cmake_policy(SET CMP0126 NEW)
+set(CMAKE_POLICY_DEFAULT_CMP0126 NEW)
+
+##############################################################################
+# - Download and initialize RAPIDS CMake helpers -----------------------------
+
+file(DOWNLOAD https://raw.githubusercontent.com/rapidsai/rapids-cmake/branch-22.06/RAPIDS.cmake
```

Comment:
I am not sure if we want to add dependency on Rapids, even if it is only in CMake. @magnatelee ?

Reply:
Why not?

Reply:
I don't see any problem. I guess we could also embed the file in this repository, instead of downloading it all the time.

Reply:
Sounds good to me, I just wanted to double-check.